Gems/Atom/RHI/DX12/Code/Source/RHI/HeapAllocator.h

```diff
@@ -73,7 +73,7 @@ namespace AZ
 
         private:
             Descriptor m_descriptor;
-            D3D12_HEAP_FLAGS m_heapFlags;
+            D3D12_HEAP_FLAGS m_heapFlags = D3D12_HEAP_FLAG_NONE;
             D3D12_HEAP_TYPE m_heapType;
```

Comment:
Does this need to be initialized too?

Reply:
It would always be assigned value in the initialize function